README.md

```diff
@@ -50,4 +50,19 @@ Using a terminal you should change directory to this project and run the followi
 vagrant will begin to build your app using gunicorn rather than flask, once the build scripts stop providing feedback you
 can now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+### Run using docker
 
+### Development 
+### build
+```docker build --target development --tag todo-app:dev . ```
+### run
+```docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)"/,target=/todo_app todo-app:dev```
```

Comment:
The bind mount is wrong here (you mean `todo-app` and not `todo_app`).

However the app won't run with this change as the `.venv` folder is copied over and is not linux compatible.

You should move your python code over to a subfolder and bind mount that.